cmd/odf/restore/crds.go

```diff
@@ -1,12 +1,54 @@
 package restore
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
 	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
 	pkgrestore "github.com/rook/kubectl-rook-ceph/pkg/restore"
 	"github.com/spf13/cobra"
 )
 
+// groupVersions defines the supported CRD groups and their corresponding API versions.
+var groupVersions = map[string]string{
+	"ocs.openshift.io":       "v1",
+	"ceph.rook.io":           "v1",
+	"storage.k8s.io":         "v1",
```

Comment:
Is this for csidrivers and storageclasses? Seems like we shouldn't need to restore this type. ODF might create resources of that type, but I don't anticipate they would have finalizers that we would need to worry about.

Reply:
removed "storage.k8s.io" CRD